packages/osd-ace/package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@osd/ace",
   "version": "1.0.0",
   "private": true,
-  "main": "./target/index.js",
+  "browser": "./target/web/index.js",
+  "main": "./target/web/index.js",
+  "types": "./target/types/index.d.ts",
   "license": "Apache-2.0",
   "scripts": {
     "build": "../../scripts/use_node ./scripts/build.js",
```

Comment:
Why do we keep the build script for this package but get rid of others?

Reply:
The dev who raised this PR was in a hurry. I have fixed it :p